english/dentist/ouch.md

```diff
@@ -1,3 +1,4 @@
 Uh-oh, you feel a dull pain in an a upper molar
 on the right side of your mouth. Shouldn't have eaten
-all those marshmallows.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.